nifi-extension-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/shared/azure/eventhubs/AzureEventHubAuthenticationStrategy.java

```diff
@@ -23,7 +23,8 @@
  */
 public enum AzureEventHubAuthenticationStrategy implements DescribedValue {
     MANAGED_IDENTITY("Managed Identity", "Authenticate using the Managed Identity of the hosting Azure resource."),
-    SHARED_ACCESS_SIGNATURE("Shared Access Signature", "Authenticate using the Shared Access Policy name and key.");
+    SHARED_ACCESS_SIGNATURE("Shared Access Signature", "Authenticate using the Shared Access Policy name and key."),
+    OAUTH2_CLIENT_CREDENTIALS("OAuth2", "Authenticate using an OAuth2 Access Token Provider backed by an Entra registered application.");
```

Comment:
Is this limited to the `Client Credentials` flow? Either the display name should read `OAuth2 Client Credentials`, or the enum should be changed, since the Token Provider can be configured with other types.